Makefile

```diff
@@ -271,3 +271,17 @@ localnet-stop:
 	docker-compose down
 
 .PHONY: localnet-start localnet-stop
+
+###############################################################################
+###                                Images                                   ###
+###############################################################################
+
+build-operator-img-all: build-operator-img-core build-operator-img-node
+
+build-operator-img-core:
+	docker-compose -f contrib/terra-operator/docker-compose.build.yml build core --no-cache
+
+build-operator-img-node:
```

Comment:
This command will most likely fail if its called without the core image present on the docker host. So it might be good to add some sort of check to ensure it doesnt try if the core image is not available?